### PR TITLE
updated dataloader

### PR DESCRIPTION
**Pull request by UBCMark**

### CODE REVIEW

File: data_loader.py [hunk 1]
```diff
@@ -9,13 +9,14 @@
...
+import h5py
 # import matplotlib.pyplot as plt
 
 
 class CocoDataset(data.Dataset):
     """COCO Custom Dataset compatible with torch.utils.data.DataLoader."""
-    def __init__(self, root, coco_annotation, vocab, coco_detection_result, 
-                 transform=None, yolo=False, dummy_object=0):
+    def __init__(self, root, coco_annotation, vocab, MSCOCO_result, coco_detection_result,
+                 transform=None, yolo=True, dummy_object=0):
```
Comment (TamakiSakura) [inline data_loader.py:19]: I think it's more convenient if the default is False

**Review by TamakiSakura — CHANGES_REQUESTED** [review]
What is the purpose of `data_loader_visual` file? 
Could we design `data_loader` in the way such that we can use the `yolo` flag to decide if we need visual feature or not?

File: data_loader.py (hunk 1 above)
Comment (UBCMark) [reply data_loader.py:19]: noted

**Comment by UBCMark** [comment]
`data_loader_visual` is useless, I will delete it

File: data_loader.py (hunk 1 above)
Comment (TamakiSakura) [reply data_loader.py:19]: actually nevermind I think this is ok

**Review by TamakiSakura — APPROVED** [review]

**Pull request merged**